### DISCUSSION TITLE
Dark Theme Support -

### DISCUSSION THREAD

**Discussion by arunsivaramanneo** [arunsivaramanneo/GPU-Viewer#61] [Show and tell]
Well, I have decided to experiment with the Dark theme support for GPU-Viewer and i was able to get a one running. So here is the first outlook of the application. Please do comment how it looks and what modification is neeeded. I will take that into consideration and see if i can implement it. As of now, i am not releasing the app to public

![Screenshot from 2023-01-15 23-59-52](https://user-images.githubusercontent.com/30646692/212885844-95ca87f2-0120-4579-b04c-10a3ced1b589.png)

**Comment by deathtrip** [comment]
Looks great. I use a dark theme system-wide and right now GPU-Viewer looks broken, with a mix of black and white backgrounds. And nothing is readable. So hopefully this dark theme will be integrated into the app in the near future.

**Reply by arunsivaramanneo** [reply]
thanks ... can i  know which distro are you using ?

**Reply by deathtrip** [reply]
Arch Linux with KDE and Sway as de/wm.

**Reply by arunsivaramanneo** [reply]
thanks .... i have updated the app and removed dependency to themes... it should now be able to display the content correctly in light theme ... parallely i will work on the dark theme support... if you are able to clone, install and test it that would be great

**Comment by arunsivaramanneo** [comment]
I think its almost ready now ... anyone seeing this please do let me know what you think about the dark mode

![Screenshot from 2023-02-05 21-47-12](https://user-images.githubusercontent.com/30646692/216847941-b8d1c5d5-0622-4123-850e-b8105cfebb1e.png)
